sparse/usr/bin/droid/android-links.sh

```diff
@@ -1,14 +1,24 @@
 #!/usr/bin/env bash
 
-MNT=$(grep "Where=" /lib/systemd/system/android.mount | cut -d'=' -f 2)
+MNT=$(grep "Where=" /lib/systemd/system/android.mount | cut -d'=' -f 2)/media/0
+
+#fix a stupid mistake in previous versions
+if [[ "$(readlink -- /home/nemo/Pictures/Android)" == "/android/Pictures" ]]; then
```

Comment:
Make double brackets to be single brackets, and double == to be single = (it's all a needless bashism)